### DISCUSSION TITLE
Multiple Monitor

### DISCUSSION THREAD

**Discussion by xfrozt54** [rizwansoaib/whatsapp-monitor#107] [Q&A]
It is possible to monitor multiple number?

**Comment by rizwansoaib** [comment]
it's already supported.



On Fri 20 Aug, 2021, 9:20 AM xfrozt54, ***@***.***> wrote:

> It is possible to monitor multiple number?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/107>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZCILZXBKHRKQ3GHD6TT5XGHTANCNFSM5CPP7ZRA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.
